catala/aixecar/amagar-se/amagar-se.md

```diff
@@ -0,0 +1 @@
+Un cop t'amagues al llençol et tornes a dormir. Al cap d'unes hores t'aixeques i tot torna a ser normal. T'acabes de perdre l'aventura de la teva vida per covard!
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.